src/environment/environment.dev.ts

```diff
@@ -7,6 +7,8 @@ export const environment: EnvironmentFile = {
   daysToCacheLogs: 7,
   logoutClearsTestPersistence: true,
   logsPostApiKey: '',
+  logsApiUrl: 'https://john.mes.dev-dvsacloud.uk/v1/logs',
```

Comment:
Using my API subdomain for quite a few envs. I won't block as I'm out this afternoon, but please fix!

Reply:
replaced john with dev in two envs.